src/commands/Cryptography/hash.ts

```diff
@@ -3,16 +3,25 @@ import { Args } from '@sapphire/framework'
 import * as crypto from 'crypto'
 import { Message } from 'discord.js'
 import { AstraeaCommand, AstraeaCommandOptions } from '../../lib/Structures/Command'
+import cryptoJS from 'crypto-js'
 
 @ApplyOptions<AstraeaCommandOptions>({
 	name: 'hash',
 	description: 'Hash your text in sha1',
 	detailedDescription: 'Hash your text in either sha1, sha256, sha512 or md5',
-	flags: ['sha1', 'sha256', 'sha512', 'md5', 'all']
+	flags: ['sha1', 'sha256', 'sha512', 'md5', 'all', 'ripemd160', 'ripemd']
```

Comment:
- you're offering `--ripemd160`, but it seems unused
- neither `--ripemd` nor `--ripemd160` are in the description